logging/fluentd/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM fluent/fluentd:v1.2.1-onbuild
+MAINTAINER Petr Kotas <petr.kotas@gmail.com>
+
+# This image is here to distribute the config
+# other option is to provide config via mount volume
+# which will make gocli either have:
+#  - one more option ie. `agregator-conf-dir`
+#  - will have the config path hard coded thus resulting
+#    gocli being dependent of particular folder structure
+
```

Comment:
@slintes @rmohr I am talking about this particular config file. I have provided explanation here.
I vouch for keeping the custom image. It results in much simpler setup and code.

Reply:
what about a ConfigMap with the config file(s)? No need for a hardcoded or configured host path then, and no custom image :)

Reply:
ups, too late ;)

Reply:
@rmohr @slintes sorry for the delay in answer. This fluentd is created [here](https://github.com/kubevirt/kubevirtci/pull/9/files#diff-45eddaf18034ddf5a593d826b8b9c1a7R268). I am fairly sure, I can not use the ConfigMap here. This side is outside of Kubernetes cluster.

Does this add more sense to my [question](https://github.com/kubevirt/kubevirtci/pull/9#issuecomment-406192832)?

Reply:
Oh sorry! Totally forgot that this container isn't part of the k8s cluster. In that case I'm fine with a custom image. (actually I'd prefer to put this into the cluster, but I guess we would miss some logs you're interested in then, right?)

Reply:
@petrkotas but you are not copying in anything. This seems to be really only about the `CMD`? You can override that always. exactly with that line when starting the container.

Reply:
Could you have a look at the `-i` or `--inline-config` parameter for fluentd? That seems to be exactly what we want. You can then supply any config (like the config file below) via the commandline.

Reply:
I have looked at the parameter, unfortunately it does not solve the issue. It just moves it on different location.
If I run the fluentd like this
```
fluentd -i "<system>\n log_level debug\n</system>\n<source>\n@type  forward\n@log_level error\nport  24224\n</source>\n<match **>\n@type file\npath /fluentd/log/collected\n</match>" -p /fluentd/plugins $FLUENTD_OPT -v
```
I still have to build the custom docker image to host the `CMD`. 

There is a hack I can do. I can create custom entrypoint script that will build the parameters from the environment variables and than run the fluentd command. Something like
```
#!/usr/bin/env bash

fluentd -i "${MY__FLUENTD_CONFIG}" -p /fluentd/plugins $FLUENTD_OPT -v
```
However, even in this case, I still have to build custom image to house the changes.

Therefore, I vouch for the current custom image to stay as the simplest solution.